static/panes/diff.js

```diff
@@ -114,74 +116,97 @@ function Diff(hub, container, state) {
     this.rhs = new State(state.rhs, monaco.editor.createModel('', 'asm'), state.rhsdifftype || DiffType_ASM);
     this.outputEditor.setModel({ original: this.lhs.model, modified: this.rhs.model });
 
-    var selectizeType = this.domRoot.find('.difftype-picker').selectize({
-        sortField: 'name',
-        valueField: 'id',
-        labelField: 'name',
-        searchField: ['name'],
-        options: [
-            { id: DiffType_ASM, name: 'Assembly' },
-            { id: DiffType_CompilerStdOut, name: 'Compiler stdout' },
-            { id: DiffType_CompilerStdErr, name: 'Compiler stderr' },
-            { id: DiffType_ExecStdOut, name: 'Execution stdout' },
-            { id: DiffType_ExecStdErr, name: 'Execution stderr' },
-        ],
-        items: [],
-        render: {
-            option: function (item, escape) {
-                return '<div>' + escape(item.name) + '</div>';
+    this.selectize = {};
+
```

Comment:
Can you remove one of these blank lines please? We tend not to have multiple blank lines in classes. Thanks!